scripts/generate_checksum.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+SHASUM_FILES="shasums-eksctl-anywhere"
+FILE_PREFIX="eksctl-anywhere"
+
+DARWIN_TAR="darwin-amd64.tar.gz"
+LINUX_TAR="linux-amd64.tar.gz"
+
+mkdir -p $SHASUM_FILES
+cd $SHASUM_FILES
+
+release_manifest=$(curl --silent 'https://anywhere-assets.eks.amazonaws.com/releases/eks-a/manifest.yaml' -o release.yaml)
+
+latest_release_version=$(cat release.yaml | yq e '.spec.latestVersion')
```

Comment:
If piping yaml to `yq`, we need to put a dash after the yq expression to make it read inline string.
```suggestion
latest_release_version=$(cat release.yaml | yq e '.spec.latestVersion' -)
```

Reply:
I think that changed since 4.18.1
https://mikefarah.gitbook.io/yq/

Reply:
Ah I see, we need to make sure we have a check for that then or enforce 4.18+. similar to [this](https://github.com/aws/eks-anywhere-build-tooling/blob/main/build/lib/generate_projects_list.sh#L9-L18) but not quite the same.

Also not everyone might have yq locally, so testing that edge case would be good too.